<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Control System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>851</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="connectionType">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="connectionName">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="databaseName">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>53</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Connection Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>116</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>databaseName</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>145</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>84</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Connection Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="queryTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="Queries">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPlainTextEdit" name="query">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="queryTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="execute">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QComboBox" name="QueryTypeBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>791</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="parametr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>631</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dbQuery">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>40</y>
          <width>161</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Виконати запит</string>
        </property>
       </widget>
       <widget class="QTableView" name="dbView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>791</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="insertionPanel" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>469</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="insertB">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteB">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <zorder>insertionPanel</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionUkrainian">
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
